implementations/tests_df/test_issue_github.py

```diff
@@ -0,0 +1,81 @@
+import datetime
+import unittest
+import json
+from pandas.util.testing import assert_frame_equal
+
+import sys
+sys.path.append('..')
+
+from code_df.issue_github import IssueGitHub
+
+
+def read_file(path):
```

Comment:
Can't we reuse this function? It seems pretty common for all these reference implementations, or am i wrong?

Reply:
Yes, definitely. But that would require another import. For example, grimoirelab-perceval tests has a similar function, but it is quite frequently repeated in the different backend tests.

Reply:
What do you mean by "anything else that is imported is tested"?

In any case, I don't mean importing it from other packages, but sharing between the tests.

Reply:
> What do you mean by "anything else that is imported is tested"?

I meant that, except for standard packages and libraries, the only thing the tests import are the modules in the implementations directory, which are tested. Importing a function to read files would mean that that function would go untested. 

Of course, sharing that function explicitly among tests is alright, probably using a common utils_for_tests.py file...

Reply:
Well, the function is tested by the tests themselves. If you do think it also needs a test, write it, no problem. But it doesn't make sense to have a function in many tests, which is the same function.

Reply:
Once all open pull requests regarding tests are approved and merged, I'll send a patch with this change. Thanks for the review!